### ISSUE TITLE
add freeform notes field to bookmarks

### ISSUE THREAD

**Issue by lolbinarycat** [skyjake/lagrange#485]
this would simply be an extra field that would show up when pressing "edit bookmarks", with no special purpose, except maybe showing up when listing bookmarks.

would be useful for remembering /why/ you bookmared something.

**Comment by skyjake** [comment]
I recommend using tags for this purpose. You can put free-form text in the tags field, or use some sort of keywords of your own choosing.

The tags that have a special meaning all begin with a dot character, so they shouldn’t get confused with regular text.

**Comment by skyjake** [comment]
Of course, the downside is that if you view the “Bookmarks by Tag” page, the free-form words will be there spamming the list…

**Comment by lolbinarycat** [comment]
personally, i find that solution very ugly, if anything i would just use the "name" field.  what if i want to use a word that i use as a tag, but not actually tag the bookmark as that?
